src/sections/collection/datasets-list/file-card/FileCard.module.scss

```diff
@@ -0,0 +1,74 @@
+@use 'sass:color';
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/colors.module";
+@import "node_modules/@iqss/dataverse-design-system/src/lib/assets/styles/design-tokens/typography.module";
+
+.container {
+  margin: 6px 0;
+  padding: 4px 10px;
+  border: 1px solid $dv-info-border-color;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.title {
+  display: flex;
+
+  > * {
```

Comment:
We can remove this selector that applies a margin to every child, instead as we already have a flex container, we could apply a gap of 8px, em's can also be tricky sometimes.
```
.title {
  display: flex;
  gap: 8px;
}
```